modules/swagger-codegen/src/main/resources/go/configuration.mustache

```diff
@@ -27,7 +28,8 @@ func NewConfiguration() *Configuration {
         Debug: false,
         DefaultHeader: make(map[string]string),
         ApiKey: make(map[string]string),
-        ApiKeyPrefix: make(map[string]string) ,
+        ApiKeyPrefix: make(map[string]string),
+        UserAgent: "{{#httpUserAgent}}{{{.}}}{{/httpUserAgent}}{{^httpUserAgent}}Swagger-Codegen/{{{version}}}/go{{/httpUserAgent}}",
```

Comment:
Yes, different languages have different CLI name for option. For perl, it's moduleVersion. For Ruby, it's gemVersion.

I did it that way because Perl developers would have no idea what artifactVersion is and moduleVersion should be more familiar to them, and similarly for other languages.